packages/gatsby-theme-store/src/sdk/search/useSearchFiltersFromPageContext.ts

```diff
@@ -98,13 +97,5 @@ export const useSearchFiltersFromPageContext = (
       query,
       map,
     }
-  }, [
-    search,
-    pageContext.staticPath,
-    pageContext.query,
-    pageContext.map,
-    pageContext.orderBy,
-    pageContext.selectedFacets,
-    pathname,
-  ])
+  }, [search, pageContext, pathname])
```

Comment:
Is pageContext memoized? If it's not, it will run everytime.

Reply:
hmmm this was an eslint autofix. I trusted it knew what it was doing, but I'm gonna revert to be sure